apps/src/gamelab/GameLab.js

```diff
@@ -91,6 +101,23 @@ GameLab.prototype.init = function (config) {
     }
   };
 
+  // Override p5.loadImage so we can modify the URL path param
+  if (!GameLab.baseP5loadImage) {
+    GameLab.baseP5loadImage = window.p5.prototype.loadImage;
+    window.p5.prototype.loadImage = function (path, successCallback, failureCallback) {
```

Comment:
I don't love monkeypatching p5 here.  Can't we just wrap this and call our own method?  Or is this necessary because other methods inside p5 depend on this and we want them to have the benefit of our media proxy too?

This sort of stuff makes me either want us to have our own fork of p5 or wrap the whole thing properly in an interface.